Framework/Core/test/benchmark_ASoAHelpers.cxx

```diff
@@ -595,7 +595,7 @@ static void BM_ASoAHelpersCombGenCollisionsPairsSameCategories(benchmark::State&
 
   for (auto _ : state) {
     count = 0;
-    for (auto& comb : selfCombinations("fNumContrib", 2, -1, collisions, collisions)) {
+    for (auto& comb : combinations(CombinationsBlockUpperSameIndexPolicy("fNumContrib", 2, -1, collisions, collisions))) {
```

Comment:
I would keep the old name as an alias to this.

Reply:
selfCombinations as a strictly upper alias are still here. Just in benchmarks I changed to upper for 'same index' vs general block policies comparison because there aren't two block strictly upper policies anymore, only 'BlockStrictlyUpperSameIndex'.